internal/operators/lso/manifest.go

```diff
@@ -2,13 +2,55 @@ package lso
 
 import (
 	"bytes"
+	"fmt"
 	"text/template"
+
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/versions"
+	"github.com/sirupsen/logrus"
 )
 
-func lsoSubscription() ([]byte, error) {
+func getCatalogSource(openshiftVersion string) ([]byte, string, error) {
+	v := common.GetParsedVersion(openshiftVersion)
+	catalogSourceName := fmt.Sprintf("redhat-operators-v-%s", v[1])
+	data := map[string]string{
+		"MAJOR":               v[0],
+		"MINOR":               v[1],
+		"CATALOG_SOURCE_NAME": catalogSourceName,
+	}
+
+	const catalogSource = `kind: CatalogSource
+apiVersion: operators.coreos.com/v1alpha1
+metadata:
+  name: {{.CATALOG_SOURCE_NAME}}
+  namespace: openshift-marketplace
+spec:
+  displayName: Red Hat Operators v{{.MAJOR}}.{{.MINOR}}
```

Comment:
Let's also add a ``description`` field to explain that this extra catalog-source comes from assisted-installer (otherwise it won't be clear how a ``x.y`` cluster also has a ``x.y-1`` catalog-source